src/Annotation/SapiHandlerAnnotation.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/**
+ * @file
```

Comment:
@file docblock should not be placed in files that contain namespaced classes (see https://www.drupal.org/node/2304909 and https://www.drupal.org/coding-standards/docs#file)